main.go

```diff
@@ -17,47 +17,68 @@ limitations under the License.
 package main
 
 import (
-	"flag"
+	"fmt"
 	"os"
-	"strings"
 	"time"
 
 	"github.com/rancher-sandbox/rancheros-operator/pkg/operator"
 	"github.com/rancher-sandbox/rancheros-operator/pkg/services"
 	"github.com/rancher/wrangler/pkg/signals"
 	"github.com/sirupsen/logrus"
-)
-
-var (
-	namespace  = flag.String("namespace", "cattle-rancheros-operator-system", "Namespace of the pod")
-	namespaces = flag.String("namespaces", "", "A comma separated list of namespaces to watch")
+	"github.com/urfave/cli"
 )
 
 func main() {
-	flag.Parse()
-	logrus.Info("Starting controller")
-	ctx := signals.SetupSignalContext()
-
-	if os.Getenv("NAMESPACE") != "" {
-		*namespace = os.Getenv("NAMESPACE")
+	app := &cli.App{
```

Comment:
:o any insight on why choosing this cli package? Got nothing against it, I just have no idea of advantages or disadvantages. Looks neat thought, all the config at once seems nice for smaller apps!

Reply:
I didn't felt viper/cobra was a good fit as operators usually have inputs only via env/args... and I don't see us needing a full blown config structure as we won't have many inputs... but anyhow, this is a really tiny library that I usually end up using for simple CLIs, it is less verbose then cobra/viper, but of course, has less functionalities compared to.